modules/videoio/test/test_camera.cpp

```diff
@@ -104,5 +104,42 @@ TEST(DISABLED_videoio_camera, v4l_read_framesize)
 
     capture.release();
 }
+TEST(COUNT_OF_CAMERA_FRAMES, v4l_poll)
+{
+        VideoCapture cap1(0);
+        VideoCapture cap2(2);
```

Comment:
It would be interesting to check how does sync works with high and low FPS camera setting:
```.cpp
// different FPS
EXPECT_TRUE(cap1.set(CAP_PROP_FPS, 5));
EXPECT_TRUE(cap2.set(CAP_PROP_FPS, 60));
// both low
EXPECT_TRUE(cap1.set(CAP_PROP_FPS, 5));
EXPECT_TRUE(cap2.set(CAP_PROP_FPS, 5));
// both high values
EXPECT_TRUE(cap1.set(CAP_PROP_FPS, 60));
EXPECT_TRUE(cap2.set(CAP_PROP_FPS, 60));
```